### DISCUSSION TITLE
How to use projects in mmpose for 3d multi-person human pose estimation?

### DISCUSSION THREAD

**Discussion by YaqiZhai** [open-mmlab/mmpose#3179] [Q&A]
I tried RTMW3d and 3D Human Pose Two-stage Estimation Demo,they all can only come out a single person pose estimation.So is  there any projects that can be used for  really multi-person human pose estimation?

**Comment by hariouat** [comment]
> I tried RTMW3d and 3D Human Pose Two-stage Estimation Demo,they all can only come out a single person pose estimation.So is there any projects that can be used for really multi-person human pose estimation?

did you find a solution please?

**Reply by YaqiZhai** [reply]
I add a RootNet to predict  the root  keypoint's depth,it's from https://github.com/mks0601/3DMPPE_ROOTNET_ RELEASE
